scripts/run_test.sh

```diff
@@ -67,117 +67,106 @@ function create_kubeplus(){
     pod=`kubectl get pods | grep kubeplus | awk '{print $1'}`
     while ! [[ "$result" ]]; do
         if [[ "$count" -gt $loops ]]; then
-            echo "Unable to create a postgres operator."
+            echo "Unable to create moodle operator."
             echo ""
-            echo "Operator deployer logs: "
-            kubectl logs $pod -c operator-deployer
+            echo "kube discovery logs: "
+            kubectl logs $pod -c kube-discovery-apiserver
             echo ""
-            echo "Operator manager logs: "
-            kubectl logs $pod -c operator-manager
+            echo "discovery helper logs: "
+            kubectl logs $pod -c operator-discovery-helper
             exit 1
         fi
-        result=$(kubectl -n default get pods -lapp=postgres-operator  -o jsonpath="$JSONPATH" 2>&1 | grep "Ready=True")
+        result=$(kubectl -n default get pods -lapp=moodle-operator  -o jsonpath="$JSONPATH" 2>&1 | grep "Ready=True")
         echo $result
         sleep $sleep_time
-        echo "Waiting for postgres-operator..."
+        echo "Waiting for moodle-operator..."
         kubectl get pods -n default
         echo ""
         count=$((count + 1))
     done
-    echo "Successfully deployed postgres-operator!"
+    echo "Successfully deployed moodle-operator!"
     echo ""
 }
-function create_postgres(){
-    echo "Creating postgres instance ..."
-    kubectl create -f $PROJECT_HOME/examples/postgres/postgres1.yaml
-    timeout=600
-    sleep_time=3
-    loops=$((timeout/sleep_time))
-    count=0
-    result=$(kubectl -n default get pods -lapp=postgres1  -o jsonpath="$JSONPATH" 2>&1 | grep "Ready=True")
-    pod=`kubectl get pods | grep -v postgres-operator | grep postgres | awk '{print $1}'`
-    while ! [[ "$result" ]]; do
-        if [[ "$count" -gt $loops ]]; then
-            echo "Unable to create postgres."
-            echo ""
-            echo "Operator logs: "
-            kubectl logs $pod
-            exit 1
-        fi
-        result=$(kubectl -n default get pods -lapp=postgres1  -o jsonpath="$JSONPATH" 2>&1 | grep "Ready=True")
-        echo $result
-        sleep $sleep_time
-        echo "Creating postgres ..."
-        kubectl get pods -n default
-        count=$((count + 1))
-    done
-    echo "Successfully created postgres instance!"
-    echo ""
+function test_explain() {
+    echo "Testing explain endpoint ..."
+    kubectl get --raw "/apis/platform-as-code/v1/explain?kind=Moodle"  | python -m json.tool
+    curl "http://localhost:8080/apis/platform-as-code/v1/explain?kind=Moodle"
+    if [ $? -eq 0 ]; then
+        echo "Successfully called explain endpoint!"
+        echo ""
+    else
+        echo "Unable to call explain endpoint!"
+        echo ""
+        exit 1
+    fi
+
+    kubectl get --raw "/apis/platform-as-code/v1/explain?kind=Moodle.MoodleSpec"  | python -m json.tool
+    curl "http://localhost:8080/apis/platform-as-code/v1/explain?kind=Moodle.MoodleSpec"
+    if [ $? -eq 0 ]; then
```

Comment:
Use curl -i <url>

This will return response headers. First line will be HTTP Status line. You can parse that and assert response status code is '200'.